### PR TITLE
DM-45687: Add a timer to rebuild script

### PR DESCRIPTION
**Pull request by roceb**

### CODE REVIEW

File: bin/rebuild [hunk 1]
```diff
@@ -217,8 +218,9 @@ fi
   if [[ -n $TAG ]]; then
     eups tags --clone="$BUILD" "$TAG"
   fi
-
-  echo "# BUILD $BUILD completed."
+  timer_end=$(date +%s)
+  RUNTIME=$((timer_end-timer_start))
+  echo "# BUILD $BUILD completed. Took $RUNTIME seconds to complete"
```
Comment (timj) [inline bin/rebuild:223]:
```suggestion
  echo "# BUILD $BUILD completed in $RUNTIME seconds."
```
I think I prefer not to say "complete" twice.

**Review by timj — APPROVED** [review]
Looks good. Minor tweaking of sentence requested.

File: bin/envconfig [hunk 2]
```diff
@@ -72,15 +72,6 @@ if [ "$finish" = true ]; then
   return
 fi
 
-if [[ -n $BASH_VERSION ]]; then
```
Comment (timj) [inline bin/envconfig:75]: What's this change about? It's not part of the stated goal of this commit.

Comment (roceb) [reply bin/envconfig]: Committed by mistake.

**Pull request merged**